python_modules/dagster/dagster/_core/host_representation/external_data.py

```diff
@@ -509,6 +515,99 @@ def __new__(cls, error: Optional[SerializableErrorInfo]):
         )
 
 
+class ExternalFreshnessPolicyData(ABC):
```

Comment:
A thought: how bad would it be to just `@whitelist_for_serdes` each of the actual FreshnessPolicy objects and make them NamedTuples?

Reply:
Hmm good question. I can't think of a problem.  Maybe @alangenfeld can?

Reply:
yea i don't have full context but i assume these policy objects are declarative little data objects that would be fine as serdes tuples.